mods/fullpotential/scripts.js

```diff
@@ -73,6 +73,9 @@ exports.BattleScripts = {
 		if (move.useTargetOffensive) {
 			attackStat = category === 'Physical' ? 'atk' : 'spa';
 		}
+		if (move.id === 'confusion') {
```

Comment:
`confusion` is already a move id. Why not just add the id `confused` to the confusion damage check at the top of this file and forget about overriding statuses.js altogether?